Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
+echo "Updating system"
+sudo yum -y install epel-release
+sudo yum -y update
+sudo yum install -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+#Install Docker
+echo "Installing Docker"
+sudo yum -y install docker docker-ce docker-ce-cli containerd.io
+which docker
+if [[ $? -eq 0 ]]; then
+  echo "Docker installation succeeded"
+else
+  echo "Docker installation failed"
+fi
+
+#For Jenkins Installation:
+#Install Java:
+echo "Installing java..."
+sudo yum -y install java-1.8.0-openjdk.x86_64
```

Comment:
pay attention in line #14 you have installed packages without pinned version, while here the java package version is pinned. Any reason to use a mix?